src/rust-crypto/rust-crypto.ts

```diff
@@ -1040,15 +1040,12 @@ export class RustCrypto extends TypedEventEmitter<RustCryptoEvents, RustCryptoEv
      * @param event - live event
      */
     public async onLiveEventFromSync(event: MatrixEvent): Promise<void> {
-        // Ignore state event
-        if (event.isState()) return;
+        // Ignore state event or remote echo
+        if (event.isState() || event.getUnsigned().transaction_id === null) return;
```

Comment:
why would a remote echo have a transaction ID of `null` ?

Reply:
I changed, it was a misunderstanding of mine